<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VobSubInputProcessDialog</class>
 <widget class="QDialog" name="VobSubInputProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VobSub Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="3,0">
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Enter character(s)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="contTextStyle" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="symbolLabel">
           <property name="text">
            <string>S&amp;ymbols:</string>
           </property>
           <property name="buddy">
            <cstring>symbolCount</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="symbolCount">
           <property name="toolTip">
            <string>Up/Down: change value</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="styleBold">
           <property name="toolTip">
            <string notr="true">Ctrl+B</string>
           </property>
           <property name="icon">
            <iconset theme="format-text-bold">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="styleItalic">
           <property name="toolTip">
            <string notr="true">Ctrl+I</string>
           </property>
           <property name="icon">
            <iconset theme="format-text-italic">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="styleUnderline">
           <property name="toolTip">
            <string notr="true">Ctrl+U</string>
           </property>
           <property name="icon">
            <iconset theme="format-text-underline">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="contText" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-ok-apply">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="grpNavButtons" native="true">
        <layout class="QVBoxLayout" name="layNavButtons">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnPrevImage">
           <property name="toolTip">
            <string>Ctrl+Shift+Left</string>
           </property>
           <property name="text">
            <string>Prev Image</string>
           </property>
           <property name="icon">
            <iconset theme="go-previous">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPrevSymbol">
           <property name="toolTip">
            <string>Ctrl+Left</string>
           </property>
           <property name="text">
            <string>Prev Symbol</string>
           </property>
           <property name="icon">
            <iconset theme="go-previous">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNextSymbol">
           <property name="toolTip">
            <string>Ctrl+Right</string>
           </property>
           <property name="text">
            <string>Next Symbol</string>
           </property>
           <property name="icon">
            <iconset theme="go-next">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNextImage">
           <property name="toolTip">
            <string>Ctrl+Shift+Right</string>
           </property>
           <property name="text">
            <string>Next Image</string>
           </property>
           <property name="icon">
            <iconset theme="go-next">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbort">
        <property name="text">
         <string>Abort</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-cancel">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpPreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Subtitle Image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="subtitleView">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color:#000; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="format">
         <string notr="true">%v/%m</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>symbolCount</tabstop>
  <tabstop>styleBold</tabstop>
  <tabstop>styleItalic</tabstop>
  <tabstop>styleUnderline</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnAbort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
